tests/kernel/fpu_sharing/generic/testcase.yaml

```diff
@@ -21,21 +21,21 @@ tests:
     tags: fpu kernel
     timeout: 600
   kernel.fpu_sharing.generic.riscv32:
-    extra_args: PI_NUM_ITERATIONS=500
+    extra_args: PI_NUM_ITERATIONS=500 DISABLE_INT_TEST=1
     filter: CONFIG_CPU_HAS_FPU
     arch_allow: riscv32
     tags: fpu kernel
     timeout: 600
     min_ram: 16
   kernel.fpu_sharing.generic.riscv64:
```

Comment:
Not the fault of this patch, but this architecture code seems curious to me.  We're splitting an essentially identical test out N ways for every architecture and then filtering each one, that's a lot of work being handed to twister for no good reason.

Seems like this could be cleaned up with some judicious use of "#if defined(ARCH_xxxx)" and "ztest_test_skip()".